packages/cli/internal/pkg/cli/zipfile/compress_to_tmp.go

```diff
@@ -28,19 +26,15 @@ func CompressToTmp(srcPath string) (string, error) {
 
 func writeToZipRecursive(writer *zip.Writer, rootPath string) error {
 	// Expand home directory path
-	expandedRootPath, err := osutils.ExpandHomeDir(rootPath)
-	if err != nil {
-		return err
-	}
-	return filepath.WalkDir(expandedRootPath, func(currentPath string, dirEntry fs.DirEntry, err error) error {
+	return filepath.WalkDir(rootPath, func(currentPath string, dirEntry fs.DirEntry, err error) error {
```

Comment:
We are now moving the directory from ~/work/agc/amazon-genomics-cli/examples/demo-wdl-project/workflows/hello to /tmp/workflow_xxxxxxx which is why we don't need to unpack the url anymore since we have copied the underlying data. We need to do that as we need to modify the input files so that they are references to S3 so that the child nodes can download them.